docs/about/case-studies/README.md

```diff
@@ -4,6 +4,29 @@ hide:
 ---
 
 # Knative Case Studies
+
+<style>
+
+  @media (max-width: 768px) {
```

Comment:
Can we make this width a css property variable so that this can become somthing like:
```css
@media (max-width: var(--med-screen-width)) {
```

Reply:
I tried, but I learned that CSS custom properties (variables) are not supported inside media query conditions.

Reply:
oh interesting, good to know!